mod/01_rating/set/type/metric.rb

```diff
@@ -16,6 +17,64 @@
   def view_caching?
     true
   end
+
+  view :value_type_edit_modal_link do |args|
+    value_type_card = card.fetch trait: :value_type, new: {}
+    subformat_card = subformat(value_type_card)
+    text =
+      if value_type_card.new?
+        'Update Value Type'
+      else
+        subformat_card.render(:shorter_pointer_content)
+      end
+    edit_args = {
+      path_opts: {
+        slot: {
+          hide: 'title,header,menu,help,subheader',
+          view: :edit, edit_value_type: true
+        }
+      },
+      html_args: {
+        class: 'btn btn-default slotter'
+      },
+      text: text
+    }
+    render_modal_link(args.merge(edit_args))
+  end
+
+  view :short_view do |_args|
```

Comment:
this should probably be called something like "detail_view" and should be in right/value_type